changelogs/fragments/facts_vlan_issue.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - facts won't break for devices which have no support for vlan.
```

I don't know if this is a complete description of the changes, or if there's something I've missed or gotten wrong, but I feel like this could use some more specific changelog entries.
```suggestion
  - ios_facts - fix for devices which have no support for VLANs, such as L3 devices.
  - ios_vlans - fail with an error when target device does not support VLAN resource.
```

---

Oh, the second one should probably also say `instead of $(previous behavior)`